vllm/core/block/prefix_caching_block.py

```diff
@@ -293,10 +296,28 @@ def clear_copy_on_writes(self) -> Dict[BlockId, List[BlockId]]:
         """
         return self._cow_tracker.clear_cows()
 
-    def mark_blocks_as_computed(self) -> None:
-        """Mark blocks as computed, used in prefix caching."""
-        # TODO Track computed blocks.
-        pass
+    def mark_blocks_as_accessed(self, block_ids: List[int],
+                                now: float) -> None:
+        """Mark blocks as accessed, used in prefix caching."""
+
+        for block_id in block_ids:
+            if block_id in self._blocks:
+                self._blocks[block_id].last_accessed = now
+
+    def mark_blocks_as_computed(self, seq_block_ids: List[List[int]]) -> None:
+        """Mark immutable blocks as computed, used in prefix caching.
+
+        Only mark those immutable block as computed.
```

Comment:
how does this check if a block is immutable? seems it doesn't

Reply:
Another thought for this mark_blocks_as_computed is whether we could deem those blocks already has content_hash as computed one?

If so, mark_blocks_as_computed could be made as empty. To check it is computed, just need a property to check whether it has a valid content_hash